### DISCUSSION TITLE
v5.1.3-beta5

### DISCUSSION THREAD

**Discussion by alenteria** [AdoPiSoft/Beta-Releases#121] [Ideas]
<hr /><em>This discussion was created from the release <a href='https://github.com/AdoPiSoft/Beta-Releases/releases/tag/v5.1.3-beta5'>v5.1.3-beta5</a>.</em>

**Comment by reydeyes** [comment]
in messenger chat you said. you did a re-upload of beta 5. but why is it still 1 week ago.

**Reply by alenteria** [reply]
last upload is yesterday https://github.com/AdoPiSoft/Beta-Releases/releases/tag/v5.1.3-beta5

**Comment by verniegithub** [comment]
Just to let you know that the latest upload today have a tendency to open the portal on the main browser with all other tabs in it. This is not the case with previous versions.  In some cases it did open on a separate browser like the usual behavior. I use the Captive portal app to open the portal. It's the same app used with the previous versions.

**Comment by verniegithub** [comment]
On my test setup with TB74 bill acceptor, bill is accepted but no credit is given (could be pulse is not read correctly?). 
Bill acceptor setting is 1 pulse = P10. Bill acceptor is activated together with the coin slot using a relay. Coin slot works fine when coin is inserted and give credit. Using another software, same hardware setup, credit is given for inserted bill.  There were versions of ado before that this TB74 works but at some point after some upgrades no credit is given. I stopped testing since then until now. 

TB74 was set to Fast Output Pulse where Lo=5ms and Hi=100ms. This hardware setup is confirmed stable already in terms of wiring the fact that two other vendo software works well with the TB74  and coin slot but I do not wish to use them as they are behind in terms of capability to handle random mac address, no eloading option is available with their setup and no option for pulse setting but can be  edited on the code if I have to change it.

**Comment by verniegithub** [comment]
v5.1.3-beta6 
- still present ang issue na ang credit ng user mapunta sa iba.

**Reply by reydeyes** [reply]
True. naka disable na yung mac sychronizer ganuna pa din

**Reply by alenteria** [reply]
paki delete ng affected device records, baka naka-sync na yan sila before sa update.

**Reply by ZaIjEn20** [reply]
sa newly flashed q n mini pc ok nmn beta3 nga lang gamit q...

**Reply by verniegithub** [reply]
> paki delete ng affected device records, baka naka-sync na yan sila before sa update.

Ganyan palagi ang ginagawa ko ang mag delete ng device records sa affected device and also clear cache and cookies from the cellphone browser ng user.  Dumaan na rin ng factory reset ang affected units.  Over time, same issue will happen again on the same phone unit. 

It's been two weeks wala na ang ganitong issue after deleting  every time mangyari ito. So wala nang problema sa running system pero after pag upgrade last midnight andun agad ang problema. Deleting again and it is ok for now. 

Another issue is on the portal theme, it took more than once na reset, clear cache then edit para maging stable.

On the plus side, I noticed improved performance sa system ng ado and thank you for that.

**Comment by ZaIjEn20** [comment]
sana po my ISO file dn itong beta6 pra direct n kmi mgfflash tulad nung sa beta2..

**Comment by markvinb101** [comment]
beta 6 pc version may issue sa wireless sub vendo with relay, kapag ini insert coin hindi nag tu turn on ang coinslot. kaya balik ako sa beta 3 at naging okay.

**Reply by markvinb101** [reply]
btw okay na pala yung sa wireless sub vendo na hindi nag to turn on ang coinslot pag nag insert coin. ni reflash ko na lang, at yun okay na. not sure kung bugs lang ba ata cguro yun.

**Comment by reydeyes** [comment]
please upload auto install for RPI4 and OPI

**Comment by markvinb101** [comment]
beta 6 issue kapag ini hard restart or brownout pag boot up, nag lo-loop reload palagi ang portal tapos may naka lagay "Unable to sync device information. The page will have to reload". Pero pag soft restart, then pag boot up nagiging normal na yung pag labas ng portal at sa pag reload. Nakaka hasle lang pag nag brown out dito sa area namin. kapag bumalik na ang kuyente, kailangan ko pa e open ang admin para mag soft restart kasi bumbalik sya sa normal sa paglabas ng portal. basta yung pag hard restart or nag powerdown dito samen bumabalik yung issue nya na laging nag re reload ang portal. btw im using pc version

**Reply by alenteria** [reply]
pahingi po ng system logs. Paki attach dito

**Reply by markvinb101** [reply]
[system-report.txt](https://github.com/AdoPiSoft/Beta-Releases/files/11394483/system-report.txt)

**Reply by markvinb101** [reply]
btw meron akong 2 wireless sub vendo, minsan talaga yung pag tap insert coin ayaw mag turn on ng coinslot kaya hindi na huhulogan ng coins kelangan ko pa e cancel then tap again insert coin at yun ng turn on na  ang coinslot at pwede na mahulogan.

**Reply by cybermind2** [reply]
> beta 6 issue kapag ini hard restart or brownout pag boot up, nag lo-loop reload palagi ang portal tapos may naka lagay "Unable to sync device information. The page will have to reload". Pero pag soft restart, then pag boot up nagiging normal na yung pag labas ng portal at sa pag reload. Nakaka hasle lang pag nag brown out dito sa area namin. kapag bumalik na ang kuyente, kailangan ko pa e open ang admin para mag soft restart kasi bumbalik sya sa normal sa paglabas ng portal. basta yung pag hard restart or nag powerdown dito samen bumabalik yung issue nya na laging nag re reload ang portal. btw im using pc version

na encounter ko na rin po ito pero doon sa first release ng beta 6, hindi yung last release...2 days ago.

**Comment by ZaIjEn20** [comment]
Sna po may option na sa pgcreate/generate ng voucher code kung kaninong coinslot/subvendo/vendo ita-tag or assign pra kpg ng-inventory for profit sharing madali nlng po ang pgcocompute..super hassle kc dhl need m ilista isa2 ung mga voucher n nagamit pra malaman m kung kanino idadagdag ung sales..pero ung voucher na nka-tag sa isang vendo/coinslot ay gagana p rin sa buong system pra kht mgpalipat-lipat p sila ng vendo dala p rin nila ung oras nila...malaking tulong po ito sa mga katulad q n my multiple subvendos..tnx po admin..sna mapansin m po ung request q....sensya n kung demanding..

**Comment by russelxrussel** [comment]
Hello sir, pwede po kaya makapag request sa next beta update nyo if possible lagyan ng (date of activate or used) at (voucher code used) sa device activity history.

please see attachment. Thank you.
![request_ado_101](https://user-images.githubusercontent.com/19811609/236406024-8480bf60-879e-4d45-8d4d-fc44483e42ad.png)

**Comment by hasap86** [comment]
v5.1.3-beta6 Lite version please :)

**Comment by Msheke** [comment]
In running Beta6 on RPI 3, machine is working okay but I can't access the Captive Portal Theme settings on admin page. See error on pic 
![Screenshot_20230520_204049_com android chrome_edit_3503456947901](https://github.com/AdoPiSoft/Beta-Releases/assets/107319095/ab20adb6-78f5-4cd3-be85-3d0670cb4ba5)

**Reply by hasap86** [reply]
Clear Portal Cache

**Comment by julogs16** [comment]
Daming Bug no.1 bug sa passcode  nag duduplicate 
no.2 bug Duplicate parin sa Sales!
No. 3 Bug may time sa server pero disconnect sa tomer
No. 4 pag naka enable c passcode bagong insert coin pag done na wala ang time yun pla nag duplicate .


![350203274_2492577894245527_4043238858725471802_n](https://github.com/AdoPiSoft/Beta-Releases/assets/134938112/81711fee-d866-4adf-a90a-eb95e266f40d)
![349371933_955122892572908_6151411719552706619_n](https://github.com/AdoPiSoft/Beta-Releases/assets/134938112/3f6a01a2-eae4-4492-bb46-043c35eded51)
